CHANGELOG.md

```diff
@@ -21,6 +21,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Deprecated
 
+## [2.39.3] - 2023-06-07
+
+### Add
+
+- For ExtDataDriver.x only, added logging config to Tests/ExtDataDriverMod.F90 to enable Logger there
```

Comment:
@metdyn You don't need to add a new version number or anything. Rather, you should add your changelog entries in the `## [Unreleased]` section at the top of the CHANGELOG.

We accumulate them there and then when it's time for a release, we put them all under a version number and date.

Reply:
I took care of it due to an "Update branch" conflict